src/buttons/modTicket.ts

```diff
@@ -0,0 +1,86 @@
+import {
+  ButtonInteraction,
+  ChannelType,
+  Client,
+  EmbedBuilder,
+  TextChannel,
+} from 'discord.js'
+import { channelIds, roleIds } from '../globals'
+import { emoji } from '../utils/emojis'
+import { errorEmbed, successEmbed } from '../utils/embeds'
+
+export const button = {
+  name: 'modTicket',
+}
+
+export const execute = async (
+  _client: Client,
+  interaction: ButtonInteraction
+) => {
+  if (!interaction.inCachedGuild()) return
+
+  const auctionsTickets = interaction.client.channels.cache.get(
```

Comment:
might be worth to use interaction.client.channels.fetch() instead as this will return null if the channel is not already in the cache. However, this is _unlikely_ to cause errors. Just leaving this here as a fyi